src/Integration.Vsix/SonarLintTagger/IssueTagger.cs

```diff
@@ -82,8 +82,8 @@ public IEnumerable<ITagSpan<IErrorTag>> GetTags(NormalizedSnapshotSpanCollection
             }
 
             return issueMarkers
-                .Where(marker => spans.IntersectsWith(marker.Span))
-                .Select(marker => new TagSpan<IErrorTag>(marker.Span, new ErrorTag(PredefinedErrorTypeNames.Warning, marker.Issue.Message)));
+                .Where(marker => marker.Span != null && spans.IntersectsWith(marker.Span.Value))
```

Comment:
🤮

Reply:
There's no point in passing non-taggable markers to the issue tagger.
The code in the tracker that calculates the affected span has does have to care about which issues are taggable, so it could be responsible for just passing taggable issues to the issue taggers.